app/main/views/organisations.py

```diff
@@ -156,10 +158,51 @@ def organisation_dashboard(org_id):
         **{
             f'total_{key}': sum(service[key] for service in services)
             for key in ('emails_sent', 'sms_cost', 'letter_cost')
-        }
+        },
+        download_link=url_for(
+            '.download_organisation_usage_report',
+            org_id=org_id,
+            selected_year=year
+        )
     )
 
 
+@main.route("/organisations/<uuid:org_id>/download-usage-report.csv", methods=['GET'])
+@user_has_permissions()
+def download_organisation_usage_report(org_id):
+    selected_year = request.args.get('selected_year')
+    services_usage = current_organisation.services_and_usage(
+        financial_year=selected_year
+    )['services']
+
+    column_names = OrderedDict([
+        ('service_id', 'Service ID'),
+        ('service_name', 'Service Name'),
+        ('emails_sent', 'Emails sent'),
+        ('sms_billable_units', 'SMS fragments sent'),
+        ('sms_remainder', 'Free SMS fragments left'),
+        ('sms_cost', 'Spent on SMS'),
```

Comment:
I know the concepts here are tricky but I’d prefer that we avoid ‘SMS’ and ‘fragments’.

‘SMS’ because we try to say ‘text messages’ (except in our technical documentation and client libraries).

‘Fragments’ because it’s a new term and it doesn’t quite mesh with how we talk about long messages being ‘charged as more than one message‘.

My attempt, cherry-picking language from elsewhere would be:
```suggestion
        ('sms_billable_units', 'Text messages charged'),
        ('sms_remainder', 'Free text message allowance remaining'),
        ('sms_cost', 'Spent on text messages'),
```

I’d be keen to know what @karlchillmaid thinks too.

Reply:
I am ok to avoid SMS, but `"text messages charged"` sounds really unclear to me. They are not charged for all those messages, as some will be in free allowance, and I think it would be good to indicate about fragments, even if historically we didn't do it much.

Reply:
Oof yeah. It really is tricky.

Maybe we just lose the `sms_billable_units` column for now until we have a better way of explaining this stuff? We don’t show this information on the page now.

Reply:
Could do, although I'd wait for @karlchillmaid to weigh in, I will link him this PR :)

Reply:
I agree with Chris that – in order to get this PR merged – we might be better leaving off the title of the `sms_billable_units` column as ‘Sent’ for now.

We know that this is confusing because:
* the figures on the dashboard and usage page don’t tally
* the labelling on both of those pages does not make the distinction clear
* the Pricing page does not work well enough, or users aren’t finding that information when they need it
* we haven’t done the hard work to establish some clear language and definitions around this stuff

I’d like to see us prioritise this piece of work asap.

Reply:
What if the page showed the number of whole text messages sent and text message fragments sent.

Reply:
@servingUpAces yeah, I’d like it to do that – I think it makes a lot of sense. If we do this (and I hope we do) we still need to get the language around those 2 things clear and consistent across the rest of Notify too.

Reply:
> we might be better leaving the title of the sms_billable_units column as ‘Sent’ for now

Just `Sent`? But then how do they know if it's letters, or sms fragments, or full sms or what? Or do you mean cutting this column out completely?

> I’d like to see us prioritise this piece of work asap.

I agree, it's a pretty important one.

> What if the page showed the number of whole text messages sent and text message fragments sent.

I'd like that, too, but I feel it's a separate story as we would have to alter the query for that, and if we do that we may want to think what other things to include.

Reply:
Sorry, yes ‘Text messages sent’ – to be consistent with the emails and letters columns.
I could have been clearer about that.

Reply:
Hey @karlchillmaid after more discussion with Ben I am leaning towards getting rid of this column altogether for now. We can think about reintroducing it later after we have a strategy for presenting this data.

The reason I am leaning this way is that the value in this column is: the number of sms fragments multiplied by international weighing. Meaning if someone sends a 2-fragment message abroad to a country with x2 multiplier, it would add 4 units to this value, if that makes sense. We currently don't have a good way of relaying this to our users.

Thinking about how other companies, like mobile networks, do this, I would say "credits" is the right word, for example "sms credits used". At present we don't use this term anywhere else though.